fury/primitive.py

```diff
@@ -626,6 +626,55 @@ def prim_triangularprism():
     return vertices, triangles
 
 
+def prim_pentagonalprism(height=1.0):
+    """Return vertices and triangles for a pentagonal prism.
+
+    Returns
+    -------
+    vertices: ndarray
+        vertices coords that compose our prism
+    triangles: ndarray
+        triangles that compose our prism
+
+    """
+    # Local variable to represent the square root of five
+    five = math.sqrt(5)
+    onec = (five - 1) / 4.0
+    twoc = (five + 1) / 4.0
+    sone = (math.sqrt(10 + (2 * five))) / 4.0
+    stwo = (math.sqrt(10 - (2 * five))) / 4.0
+
+    vertices = np.array([[stwo/2, twoc/2, -1.84],
+                         [sone/2, -onec/2, -1.84],
+                         [0, -1/2, -1.84],
+                         [-sone/2, -onec/2, -1.84],
+                         [-stwo/2, twoc/2, -1.84],
+                         [stwo/2, twoc/2, 1.84],
+                         [sone/2, -onec/2, 1.84],
+                         [0, -1/2, 1.84],
+                         [-sone/2, -onec/2, 1.84],
+                         [-stwo/2, twoc/2, 1.84]])
+    vertices[:, 2] = height*vertices[:, 2]
```

Comment:
pep8: Can you add spaces between multiplication here. Thank you!

Reply:
If I am removing `height` then how can I add `vertices[:, 2] = height * vertices[:, 2]` line?

Reply:
Should I consider removing `vertices[:, 2] = height*vertices[:, 2]` line?